app/code/RLTSquare/Ccq/Console/Command/HelloWorld.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+declare(strict_types=1);
+
+namespace RLTSquare\Ccq\Console\Command;
+
+use Exception;
+use Magento\Framework\MessageQueue\PublisherInterface;
+use Magento\Framework\Serialize\Serializer\Json;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * class for helloworld command
+ */
+class HelloWorld extends Command
+{
+    /**
+     * @const
+     */
+    private const VAR1 = 'var1';
```

Comment:
Naming convention - since you opted for using constants, it would have been better to go with something like `private const INPUT_OPTION_VAR1 = 'var1';. private const INPUT_OPTION_VAR2 = 'var2';.` that would emphasize the meaning more clearly.